USER_GUIDE.md

```diff
@@ -118,6 +118,35 @@ var request = new SearchRequest
 
 var searchResponse = client.Search<Tweet>(request);
 ```
+
+#### Searching for presence or absence of exact terms
+
+From the documentation on [Term-level queries](https://opensearch.org/docs/latest/query-dsl/term/):
+
+> Term-level queries search an index for documents that contain an exact search term...
+>
+> When working with text data, use term-level queries for fields mapped as keyword only.
+>
+> Term-level queries are not suited for searching analyzed text fields. To return analyzed fields, use a full-text query.
+
+The above search example includes a `TermQuery` matching documents with `user:kimchy`.
+
+Term-level queries with empty or null values however are stripped from search requests by default.
+
+To search for documents which contain a non-null but empty field value (i.e. an empty string) include an `IsVerbatim` property or a `.Verbatim()` clause like this:
+
+```csharp
+var request = new SearchRequest
+{
+    From = 0,
+    Size = 10,
+    Query = new TermQuery { Field = "user", Value = "kimchy", IsVerbatim = true } ||
+            new MatchQuery { Field = "description", Query = "OSC" }
+};
+
+var searchResponse = client.Search<Tweet>(request);
+```
```

Comment:
I feel like this a poor example, as the `IsVerbatim` is irrelevant to this query. The text discusses it's usefulness when dealing with an empty string term query, but then the code snippet is using a non-empty query.

Reply:
I also think adding the fluent variant would be helpful

Reply:
was on vacation for a week there; let me look to improve that.

Reply:
updated